docs/developer/developer-manual.html

```
@@ -172,7 +172,7 @@ <h2 id="Build_tasks">Build tasks</h2>
   <li> <code>spotlessApply</code>: reformats Java files.
   <li> <code>spotlessCheck</code>: checks formatting of Java files.
   <li> <code><em>NameOfJUnitTest</em></code>: runs the JUnit test with that name; for example, <code>NullnessTest</code>.
-  <li> <code>task</code>: lists tasks; use <code>--all</code> to see all tasks.
+  <li> <code>tasks</code>: lists tasks; use <code>--all</code> to see all tasks.
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </li></li></li></li></li></li></li></li> ], start tag match failed [ <li> ] on line 175.

(tag-pair)